www/class/centreonHost.class.php

```diff
@@ -114,6 +117,80 @@ public function getList($enable = false, $template = false, $exclude = null)
         return $listHost;
     }
 
+    /**
+     * get the template currently saved for this host
+     *
+     * @param $hostId
+     *
+     * @return array
+     */
+    public function getSavedTpl($hostId): array
+    {
+        $mTp = [];
+        $dbResult = $this->db->prepare(
```

Comment:
Joining with the host table is not necessary.

Reply:
it is required to get the templates name displayed in the select2